partner_multi_relation_parent/models/res_partner_relation.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, models
+
+
+class ResPartnerRelation(models.Model):
+    _inherit = 'res.partner.relation'
+
```

Comment:
code for not allowing more than one parent relation, for unlinking a relation, for creating a relation is missing